packages/gatsby/src/cache-dir/component-renderer.js

```diff
@@ -67,11 +77,16 @@ class ComponentRenderer extends React.Component {
 
   render() {
     if (this.state.pageResources) {
-      return createElement(this.state.pageResources.component, {
-        key: this.props.location.pathname,
-        ...this.props,
-        ...this.state.pageResources.json,
-      })
+      return createElement(
```

Comment:
@KyleAMathews can you sanity check here, I've moved the layout rendering to the component-renderer, which makes things neater, but I want to make sure I didn't overlook anything when making the move.